app/models/user_track/external.rb

```diff
@@ -10,7 +10,7 @@ def initialize(track)
       @track = track
     end
 
-    delegate :concepts, :num_concepts, :updated_at,
+    delegate :concepts, :num_exercises, :num_concepts, :updated_at,
```

Comment:
I think this might not work. 
Check the definition of `exercises`:

```ruby
def exercises
  enabled_exercises(track.exercises)
end
```

Whereas `num_exercises` contains the active _and_ inactive exercises IIRC.

Reply:
> Whereas num_exercises contains the active and inactive exercises IIRC

Interesting. I think we should change `num_exercises` to just be active + beta then. I can't see any point in us having just a total count.

Reply:
Agreed. That would be nice